improver/lapse_rate.py

```diff
@@ -383,70 +300,54 @@ def _generate_lapse_rate_array(
             numpy.ndarray
                 Lapse rate values
         """
-        # Fill sea points with NaN values. Can't use Numpy mask since not
-        # recognised by "generic_filter" function.
+        # Fill sea points with NaN values.
```

Comment:
Whilst you're here, could you stick colons on the end of lines 292, 294, 296m and 300 (after the data types in brackets).

Reply:
Done